apps/src/lab2/views/components/editor/CodeEditor.tsx

```diff
@@ -33,13 +38,46 @@ const CodeEditor: React.FunctionComponent<CodeEditorProps> = ({
   const [editorView, setEditorView] = useState<EditorView | null>(null);
   const channelId = useAppSelector(state => state.lab.channel?.id);
   const isReadOnly = useAppSelector(isReadOnlyWorkspace);
+  const editorFontSizeKey = useAppSelector(
```

Comment:
nit: instead of having two variables `editorFontSizeKey` and `fontSizeKey` we could do this instead inside the app selector
```
state => state.codebridgeWorkspace.editorFontSizeKey || DEFAULT_FONT_SIZE_KEY
```